content/docs/self-hosted-runners.md

```diff
@@ -178,13 +213,13 @@ you plan to deploy runners to. Bot accounts are
 [the same](https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts#personal-user-accounts)
 as normal user accounts, with the only difference being the intended use case.
 
-### Cloud credentials
+### Runner Credentials
```

Comment:
💯  We should add this to our CML thesaurus, after creating it, of course. 🙃

Reply:
Could also say "Compute Credentials?" Trying to distinguish from storage creds.

Reply:
Yes, Compute/Storage seems still a more appropriate dichotomy.